example/app/config.yml

```diff
@@ -81,17 +81,38 @@ default:
     subnets:
       - '{{AWS_SUBNET}}'
 
+  alarms:
+    CPUHigh:
+      alarm_action: ScaleOutScalingPolicy
+      threshold: '50'
+    CPULow:
+      alarm_action: ScaleInScalingPolicy
+      comparison_operator: LessThanThreshold
+      threshold: '50'
+
   ecs:
     instanceType: t2.medium
     desiredInstances: 1
+    maxInstances: 2
+    minInstances: 1
     availabilityZone: us-east-1b
     amiid: ami-a7a242da
     publicIp: true
     restartTasksOnDeploy: true
     docker:
       username: cumulususer
+    autoscaling_policies:
+      ScaleOut:
+        policy_type: SimpleScaling
+        adjustment_type: ChangeInCapacity
+        scaling_adjustment: 1
+      ScaleIn:
+        policy_type: SimpleScaling
+        adjustment_type: ChangeInCapacity
+        scaling_adjustment: -1
+
     services:
-      EcsTaskHelloWorld:
+      HelloWorldService:
```

Comment:
Why the name changes?

Reply:
We could probably keep this name as-is but `EcsTaskHelloWorld` could represent _either_ fargate or EC2, whereas an ECS service depends I believe on a running EC2 ECS instance. So I thought these names made more sense now that we have a distinction between the `HelloWorld` as a long running ECS Service Activity and `HelloWorld` on demand using fargate.